templates/shared/_download-ubuntu-thank-you-pro-section.html

```diff
@@ -0,0 +1,36 @@
+<section class="p-section">
+  <div class="u-fixed-width">
+    <a href="/pro">
+    {{
+      image (
+      url="https://assets.ubuntu.com/v1/2f32b5f5-Ubuntu-Pro-logo.svg",
+      alt="Ubuntu Pro",
+      width="170",
+      height="42",
+      hi_def=True,
+      loading="lazy"
+      ) | safe
+    }}
+    </a>
+    <hr class="p-rule" />
+  </div>
+  <div class="row--50-50">
+    <div class="col">
+      <h2>Get enterprise support for Ubuntu Server</h2>
+    </div>
+    <div class="col">
+      <p>Ubuntu Pro is a comprehensive subscription delivering enterprise-grade security, management tooling, and extended support for developers and organisations. Ubuntu Pro is free for personal use on up to five machines</p>
+      <hr class="p-rule" />
+      <ul class="p-list--divided u-no-padding--bottom">
+        <li class="p-list__item is-ticked">Security updates for the <a href="https://ubuntu.com/esm">full open source stack</a></li>
+        <li class="p-list__item is-ticked">Estate <a href="https://landscape.canonical.com/landscape-features">monitoring and management</a></li>
```

Comment:
```suggestion
        <li class="p-list__item is-ticked">Estate <a href="/landscape/features">monitoring and management</a></li>
```
We should probs just use the correct link instead of the redirect